content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -67,19 +73,25 @@ Next, create all the jobs with one kubectl command:
 
 ```shell
 kubectl create -f ./jobs
-job "process-item-apple" created
-job "process-item-banana" created
-job "process-item-cherry" created
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output